services/apis/fence/fenceQuestions.js

```diff
@@ -128,7 +131,12 @@ module.exports = {
    * @param {Gen3Response} expectedRes
    */
   responsesEqual(actualRes, expectedRes) {
-    expect(actualRes).to.be.a.gen3Res(expectedRes);
+    err = '';
+    try {
```

Comment:
why was this necessary?

Reply:
it was helpful to know what went wrong when a service account registration failed, but later i added a log directly in `registerGoogleServiceAccount`
so i can remove this